### DISCUSSION TITLE
auto-import suggestion missing path prefix

### DISCUSSION THREAD

**Discussion by pelissarisergioo** [hrsh7th/nvim-cmp#1825] [Q&A]
I have a structure like

```
src/
    models/
           modelA.py

```
When I'm using a class reference inside the modelA, the auto-import tool suggests the import like...

```
from models.modelA import Potatoes
```
And I would expect something like

```
from src.models.ModelA import Potatoes
```

I've been reading the documentation but nothing effective for this issue atm.

**Comment by Shougo** [comment]
Hm..  It seems LSP feature.

**Comment by poctek** [comment]
Hi! It seems that I have the same issue. Maybe you had any progress?
